internal/cli/organizations/apikeys/update.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package apikeys
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/description"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/output"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+	atlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+type UpdateOpts struct {
+	cli.GlobalOpts
+	id    string
+	desc  string
+	roles []string
+	store store.OrganizationAPIKeyUpdater
+}
+
+func (opts *UpdateOpts) init() error {
+	var err error
+	opts.store, err = store.New(config.Default())
+	return err
+}
+
+func (opts *UpdateOpts) newAPIKeyInput() *atlas.APIKeyInput {
+	return &atlas.APIKeyInput{
+		Desc:  opts.desc,
+		Roles: opts.roles,
+	}
+}
+
+const updateTemplate = "Successfully updated APIKey '{{.ID}}'.\n"
+
+func (opts *UpdateOpts) Run() error {
+	r, err := opts.store.UpdateOrganizationAPIKey(opts.ConfigOrgID(), opts.id, opts.newAPIKeyInput())
+	if err != nil {
+		return err
+	}
+
+	return output.Print(config.Default(), updateTemplate, r)
+}
+
+// mongocli iam organizations|orgs apiKey(s)|apikey(s) update <ID> [--role role][--desc description][--orgId orgId]
+func UpdateBuilder() *cobra.Command {
+	opts := new(UpdateOpts)
+	cmd := &cobra.Command{
+		Use:     "update <ID>",
+		Aliases: []string{"updates"},
+		Args:    cobra.ExactArgs(1),
+		Short:   description.UpdateOrganizationsAPIKey,
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return opts.init()
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			opts.id = args[0]
+			return opts.Run()
+		},
+	}
+
+	cmd.Flags().StringSliceVar(&opts.roles, flag.Role, []string{}, usage.Roles)
+	cmd.Flags().StringVar(&opts.desc, flag.Description, "", usage.Comment)
+
+	cmd.Flags().StringVar(&opts.OrgID, flag.OrgID, "", usage.OrgID)
```

Comment:
[nit] I'm just catching this but we may have a small ux bug here, if the flag is not set and there's no org in the config we may have an error from the client
we need to do something like we do for project id where we have a generic function that validates the project is set

https://github.com/mongodb/mongocli/blob/55cce5ce4dbf25efa84fc3301f5e42ead7a3261b/internal/cli/global_opts.go#L53-L67